### ISSUE TITLE
Image fill appears broken in 3.5.1

### ISSUE THREAD

**Issue by craigmdennis** [timuric/Content-generator-sketch-plugin#98]
When using the image paste in Sketch Version 3.5.1 (25234) it changes the background to a pattern but is completely transparent.

I've removed all plugins and the result is the same.

<img src='http://d.pr/i/18FRq+' width="500">

![image](https://cloud.githubusercontent.com/assets/725067/12727873/db4c0408-c916-11e5-9545-a78e5fe3ed70.png)

**Comment by aidil** [comment]
Same issue here.

**Comment by louismagie** [comment]
@aidil I've got the issue …

**Comment by craigmdennis** [comment]
This appears to be fixed for me with the latest release.

**Comment by louismagie** [comment]
It's good for me ! Thanks :shipit:

**Comment by antynomia** [comment]
Unfortunately it's still not working (latest version 3.5.1, removed all plugins and pasted again)

**Comment by craigmdennis** [comment]
I use Sketch Toolbox and had to uninstall the plugin, then install it again. Seemed to fix it for me.

**Comment by antynomia** [comment]
I don't see at all plugin in Sketch Toolbox. But I've repeated all steps, it seems to be working again. 

But I've observed strange thing - after copying folder to Plugins folder I see two "Content Generator" folders: one with old options (geo, persona, photos, txt) and second one with new ones (so flickr tag etc.). Only second one is working ofc. Not a big problem I can delete old folder but it makes confusion, just fyi :-)
